ocean_lib/models/data_token.py

```diff
@@ -176,11 +177,11 @@ def get_event_signature(self, event_name):
 
     def get_start_order_logs(
         self,
-        consumer_address=None,
-        from_block=0,
-        to_block="latest",
-        from_all_tokens=False,
-    ):
+        consumer_address: Optional[str] = None,
+        from_block: int = 0,
```

Comment:
Whenever `getLogs` is called, `from_block` and `to_block` should be `Optional[BlockIdentifier]` 

This occurs in several places throughout this file.